sense/loading.py

```diff
@@ -0,0 +1,125 @@
+import json
+import os
+import torch
+import yaml
+
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+with open(os.path.join(os.path.dirname(__file__), 'models.yml')) as f:  # TODO: check if this is the best way of doing this
```

Comment:
@corneliusboehm any thoughts on this?

Reply:
Apparently you can get more complex (https://stackoverflow.com/a/4060259/6611214), but this looks ok to me